workflows/sock-shop-demo/usingCmdProbe/workflow.yaml

```diff
@@ -0,0 +1,670 @@
+                    imagePullPolicy: Always
+                    args:
+                    - -c
+                    - ./experiments -name pod-delete
+                    command:
+                    - /bin/bash
+                    env:
+                    - name: TOTAL_CHAOS_DURATION
+                      value: '15'
+                    # Period to wait before and after injection of chaos in sec
+                    - name: RAMP_TIME
+                      value: ''
+                    # provide the kill count
+                    - name: KILL_COUNT
+                      value: ''
+                    - name: FORCE
+                      value: 'true'
+                    - name: CHAOS_INTERVAL
+                      value: '5'
+                    - name: LIB
+                      value: 'litmus'    
+                    labels:
+                      name: pod-delete
+      container:
+        image: lachlanevenson/k8s-kubectl
+        command: [sh, -c]
+        args:
+          [
+            "kubectl apply -f /tmp/pod-delete.yaml -n {{workflow.parameters.adminModeNamespace}} | sleep 5",
+          ]
+
+    - name: install-pod-cpu-hog
+      inputs:
+        artifacts:
+          - name: install-pod-cpu-hog
+            path: /tmp/pod-cpu-hog.yaml
+            raw:
+              data: |
+                apiVersion: litmuschaos.io/v1alpha1
+                description:
+                  message: |
+                    Injects cpu consumption on pods belonging to an app deployment
+                kind: ChaosExperiment
+                metadata:
+                  name: pod-cpu-hog
+                spec:
+                  definition:
+                    scope: Namespaced
+                    permissions:
+                      - apiGroups:
+                          - ""
+                          - "batch"
+                          - "litmuschaos.io"
+                        resources:
+                          - "jobs"
+                          - "pods"
+                          - "pods/log"
+                          - "events"
+                          - "chaosengines"
+                          - "chaosexperiments"
+                          - "chaosresults"
+                        verbs:
+                          - "create"
+                          - "list"
+                          - "get"
+                          - "patch"
+                          - "update"
+                          - "delete"
+                    image: "litmuschaos/go-runner:latest"
+                    imagePullPolicy: Always
+                    args:
+                    - -c
+                    - ./experiments -name pod-cpu-hog
+                    command:
+                    - /bin/bash
+                    env:
+                    - name: TOTAL_CHAOS_DURATION
+                      value: ''
+                    
+                    - name: CHAOS_INTERVAL
+                      value: '10'
```

Comment:
Are we really using intervals in pod-cpu and memory experiments? If we are not using we can remove it from the main CRs as well. cc: @ispeakc0de